addon/components/connect.js

```diff
@@ -38,7 +38,7 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
         }
 
         this.actions = Object.assign({},
-          this.actions, dispatchToActions(redux.dispatch.bind(redux))
+          this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
```

Comment:
@foxnewsnetwork can you clarify the part you mention ^^ `variables out` ? I have a failing local build trying to use a computed property (as it would require a "break cache" like mechanic and I can't get that to work as-is). I just wanted to learn more about 1 or 2 use cases you might have to ensure they will actually "work" once we merge it.